### ISSUE TITLE
Top logo missing

### ISSUE THREAD

**Issue by bigon** [openzipkin/zipkin#2961]
Hello,

With zipkin 2.19.2, it seems that the top logo is missing (I get a 404 from https://zipkin.io/public/img/zipkin-logo-200x119.jpg)

Shouldn't the logo be bundled in the .jar file to avoid a call to an external website?

**Comment by adriancole** [comment]
agree. @openzipkin/ui do you?

**Comment by adriancole** [comment]
(ps we should fix that 404 meanwhile)

**Comment by shakuzen** [comment]
URL has been updated, so the 404 is fixed for now. Still the question of whether the logo should be packaged with the application or not.

**Comment by jeqo** [comment]
My 2p: UI should be able to work the same way even when access to internet is closed. I'm +1 on adding logo to bundle.

**Comment by jorgheymans** [comment]
Yes totally agree with @jeqo. And those full time behind an authenticating
proxy know all about sneaky external resources like this. Not needed just
package it.

Op vr 31 jan. 2020 11:34 schreef Jorge Quilcate Otoya <
notifications@github.com>:

**Comment by jcchavezs** [comment]
I think it does not make sense to not have the logo in the UI locally. If
serving the asset over http is an issue we can always serve it as base64
but include it by all means.

fre. 31. jan. 2020, 10:28 skrev Tommy Ludwig <notifications@github.com>:

**Comment by adriancole** [comment]
ok I think we can manually copy the logo here. note there was something else that made external requests (fontawesome?), but yeah regardless of that we can copy the logo in a PR and change the link. Any volunteer?

**Comment by jeqo** [comment]
Just to be sure, this is about `zipkin-classic` right? As I can't find an external reference to logo on `zipkin-lens`
